org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/XMLSchemaDiagnosticsTest.java

```diff
@@ -552,6 +552,36 @@ public void fuzzyElementNamesWithPrefixAndNoMatch() throws Exception {
 											ca(diagnostic, te(5, 12, 5, 17, "AElement2"), te(5, 28, 5, 33, "AElement2")));
 	}
 
+	@Test
+	public void elementThatCantHaveNodesHasNodes() throws Exception {

```

Comment:
To be consistent with other test name, you should prefix the test with `cvc_complex_type_2_2`, you can rename your test with `cvc_complex_type_2_2_withText`